lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -942,7 +977,11 @@ WitnessChecker::lookupValueWitnesses(ValueDecl *req, bool *ignoringNames) {
                                        SourceLoc(),
                                        lookupOptions);
     for (auto candidate : lookup) {
-      witnesses.push_back(candidate.getValueDecl());
+      auto decl = candidate.getValueDecl();
+      if (!conformance ||
+          isMemberOperator(dyn_cast<FuncDecl>(decl), conformance->getType())) {
```

Comment:
dyn_cast returns nullptr if the cast fails. If you’re sure you have a function here, use cast<>; otherwise check the return value.

Reply:
Gotcha. I was confused by the presence of `dyn_cast_or_null` in autocompletion. 🤦🏻‍♂️